HealthcareSystem/PatientWebAppUnitTests/UnitTests/PatientControllerTests.cs

```diff
@@ -0,0 +1,102 @@
+using Backend.Service;
+using Model.Users;
+using Moq;
+using PatientWebApp.Controllers;
+using Repository;
+using System;
+using Xunit;
+using Model.Enums;
+using Microsoft.AspNetCore.Mvc;
+using Backend;
+using System.Collections.Generic;
+using PatientWebApp.DTOs;
+using PatientWebAppTests.CreateObjectsForTests;
+
+namespace PatientWebAppTests.UnitTests
+{
+    public class PatientControllerTests
+    {
+        private readonly ObjectFactory _objectFactory;
+
+        public PatientControllerTests()
+        {
+            _objectFactory = new ObjectFactory();
+        }
+        private IActivePatientRepository CreatePatientStubRepository()
+        {
+            var patientStubRepository = new Mock<IActivePatientRepository>();
+            ICreateObject newValidObject = _objectFactory.GetObject("Patient");
+            var patients = new List<Patient>();
+            patients.Add((Patient)newValidObject.createValidObject());
+
+            patientStubRepository.Setup(m => m.GetPatientByJmbg("1234567891234")).Returns(patients[0]);
+            patientStubRepository.Setup(m => m.AddPatient(new Patient()));
+
+            return patientStubRepository.Object;
+        }
+
+        private IActivePatientCardRepository CreatePatientCardStubRepository()
+        {
+            var patientCardStubRepository = new Mock<IActivePatientCardRepository>();
+            ICreateObject newValidObject = _objectFactory.GetObject("PatientCard");
+            var patientCards = new List<PatientCard>();
+            patientCards.Add((PatientCard)newValidObject.createValidObject());
+
+            patientCardStubRepository.Setup(m => m.GetPatientCardByJmbg("1234567891234")).Returns(patientCards[0]);
+            patientCardStubRepository.Setup(m => m.AddPatientCard(new PatientCard()));
+
+            return patientCardStubRepository.Object;
+
+        }
```

Comment:
Will you put this in one class or leave it here?

Reply:
I will leave it here for now.

Reply:
You can make in future branch for refactoring this, that is my proposal. As I suppose you don't have time now, and for me that it's okay.